jobserver/urls.py

```diff
@@ -337,7 +341,7 @@
     path("ui-components/", components),
     path("users/", include(user_urls)),
     path("workspaces/", yours.WorkspaceList.as_view(), name="your-workspaces"),
-    path("__debug__/", include(debug_toolbar.urls)),
+    *debug_toolbar_urls,
     path("<str:project_slug>/", include(project_urls)),
 ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Following Dave's suggestion, you could change this line for consistency, so we aren't using two different ways to add to a list. However, I can understand not wanting to make an unrelated change right now.

i.e turn:
```
] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

into:
```
    *static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
]
```

Reply:
Thanks, Lucy. I considered doing so, but I didn't want to introduce an unrelated change into the diff.